src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LambdaParameterNameCheck.java

```diff
@@ -48,21 +54,20 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of checking with this config:
+ * Code Example:
  * </p>
  * <pre>
- * public class TestClass {
- *
- *     Function&lt;String, String&gt; function1 = str -&gt; str.toUpperCase().trim();
- *
- *     Function&lt;String, String&gt; function2 = _s -&gt; _s.trim().toUpperCase(); // violation
- *
- *     public boolean testMethod(String sentence) {
- *         return Stream.of(sentence.split(" "))
- *             .map(word -&gt; word.trim())
- *             .anyMatch(Word -&gt; "in".equals(Word)); // violation
- *     }
+ * class MyClass {
+ *   Function&lt;String, String&gt; function1 = str -&gt; str.toUpperCase().trim(); // OK
+ *   Function&lt;String, String&gt; function2 = _s -&gt; _s.trim().toUpperCase();
+ *           // violation, name '_s' must match pattern '^[a-z]([a-zA-Z]+)*$'
  *
+ *   public boolean myMethod(String sentence) {
+ *     return Stream.of(sentence.split(" "))
+ *             .map(word -&gt; word.trim()) // OK
+ *             .anyMatch(Word -&gt; "in".equals(Word)); // violation, name '_s'
```

Comment:
name of the parameter in error message is incorrect, looks like it was copypastedfrom previous example.

Reply:
done